apisix/schema_def.lua

```diff
@@ -496,6 +496,13 @@ _M.route = {
         },
 
         id = id_schema,
+
+        status = {
+            description = "route status, 1 to enable, 0 to disable",
+            type = "integer",
+            enum = {1, 0},
```

Comment:
add new test case: specify an invalid status value, eg: `100`

Reply:
add it